src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -3746,6 +3746,73 @@ protected SELF satisfiesExactlyInAnyOrderForProxy(Consumer<? super ELEMENT>[] re
     return myself;
   }
 
+  /**
+   * Verifies that there is exactly one element of the iterable elements that satisfies the {@link Consumer}.
```

Comment:
`of the iterable elements` -> `of the array under test`

Reply:
BTW thanks for having javadoc for the different type where `satisfiesOnlyOnce` was added!